src/screens/Analytics/GlobalSearch/GlobalSearchBarUtils.res

```diff
@@ -0,0 +1,277 @@
+let globalSeacrchAtom: Recoil.recoilAtom<GlobalSearchTypes.defaultResult> = Recoil.atom(.
+  "globalSearch",
+  {
+    GlobalSearchTypes.local_results: [],
+    remote_results: [],
+    searchText: "",
+  },
+)
+
```

Comment:
can we move this recoil definition under this file src/Recoils/HyperswitchAtom.res